ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -557,6 +575,16 @@ private boolean copyDataByIndexSegment(LogSegment logSegmentToCopy, IndexSegment
     return copiedAll;
   }
 
+  /**
+   * Determines if {@code copyCandidate} belongs to the DELETED_IN_PROGRESS {@link Container}s.
+   * @param copyCandidate the {@link IndexEntry} to check
+   * @param selectedContainerPairSet the collection of AccountId and ContainerId pairs in the given status.
+   */
+  private boolean isDeleteRequested(IndexEntry copyCandidate, Set<Pair<Short,Short>> selectedContainerPairSet) {
```

Comment:
How about renaming the method to `isFromDeletedContainer`?

Reply:
Updated.